tests/cloudformation/checks/resource/aws/test_IAMRoleAllowsPublicAssume.py

```diff
@@ -0,0 +1,32 @@
+import os
+import unittest
+
+from checkov.cloudformation.checks.resource.aws.IAMRoleAllowsPublicAssume import check
+from checkov.cloudformation.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+
+class TestIAMRoleAllowsPublicAssume(unittest.TestCase):
+
+    def test_summary(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_IAMRoleAllowsPublicAssume"
+        report = runner.run(root_folder=test_files_dir,runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        for record in report.failed_checks:
+            self.assertEqual(record.check_id, check.id)
+        
+        for record in report.passed_checks:
+            self.assertEqual(record.check_id, check.id)
+
+        self.assertEqual(summary['passed'], 2)
+        self.assertEqual(summary['failed'], 2)
+        self.assertEqual(summary['skipped'], 0)
+        self.assertEqual(summary['parsing_errors'], 0)
```

Comment:
Hi @njgibbon !

To avoid inversion of logic (happened in the past...), can you set different cluster identifiers for the passed & failed, and check that the failed resources in the report should have failed and the passed resources should have passed?

See a recent example in #1154

Reply:
Hey @njgibbon  - Ah yes, I like this! Will get on to it in a bit.